src/controller/controller.ts

```diff
@@ -576,6 +582,14 @@ class Controller extends events.EventEmitter {
         device.implicitCheckin();
         device.linkquality = dataPayload.linkquality;
 
+        // Process gpDevice as actual device in event of encapsulated notification via unicast
+        if (gpDevice) {
+            gpDevice.updateLastSeen();
+            gpDevice.implicitCheckin();
```

Comment:
I don't think we have to call implicit checking for gp devices, this is only relevant for devices where commands can be send to.

Reply:
I can remove this, no problem.